### ISSUE TITLE
Checkout Missing Alt Attribute

### ISSUE THREAD

**Issue by onur-CK** [onur-CK/BookLand#41]
**Checkout Missing Alt Attribute**
   - Description: The country select flag image on checkout page lacks an alt attribute, causing accessibility errors.
   - Add proper alt attribute to the country select flag image.
